src/python_be.cc

```diff
@@ -662,6 +675,8 @@ ModelInstanceState::ExecuteBLSRequest(
         for (auto& input_tensor : infer_request->Inputs()) {
           if (!input_tensor->IsCPU()) {
 #ifdef TRITON_ENABLE_GPU
+            // Attempt to use the cuda shared memory pool for GPU tensor.
+            ShareCUDAMemoryPool(input_tensor->MemoryTypeId());
```

Comment:
I was wondering would it be possible to initially get all the memory pool addresses and make it part of the `IPCControl` struct. Then, whenever the stub wants to use a memory pool it would check whether it exists in the map of the memory pools it has. If not it would initialize it. This would remove the need for IPC transfer for every inference and improve performance.

Reply:
Resolved offline.